### PR TITLE
Add several tests for AddWMS behaviour

### PR DESCRIPTION
**Pull request by marcjansen**
This adds a couple of tests for the AddWMS form. Please review.

### CODE REVIEW

**Comment by dnlkoch** [comment]
Do you see any chance to continue work on this @marcjansen?

**Comment by marcjansen** [comment]
Nice question. Hrm, not within the next 2-3 weeks, honestly. Feel free to close if this annoys you.

**Comment by marcjansen** [comment]
Ping @dnlkoch, This is ready for review. Thanks for the push, @dnlkoch.

File: test/spec/view/form/AddWms.test.js [hunk 1]
```diff
@@ -119,6 +119,179 @@ describe('BasiGX.view.form.AddWms', function() {
...
+                        window.setTimeout(function() {
+                            checkedCheckboxes = form.query(selPrefix + 'checkbox[checked=true]');
+                            addToMapBtn = form.down('button[name="add-checked-layers"]');
+                            expect(checkedCheckboxes.length).to.be(1);
+                            expect(addToMapBtn.isDisabled()).to.be(false);
+
+                            done();
+                        }, 50);
+                    }, 50);
+                }, 50);
```
Comment (dnlkoch) [inline test/spec/view/form/AddWms.test.js]: This looks odd to me. Maybe we should test specific functions here to avoid using of (nested) `setTimeout()`s.

**Comment by marcjansen** [comment]
Sorry for the resolving of the discussion... wrong button.

I habe specifically added these, it might be these also work wo/ this safety measures

I wanted to explicitly check the behaviour. Methods have their own block right below. The components react to things and these tests try to cover those changes.

File: test/spec/view/form/AddWms.test.js (hunk 1 above)
Comment (marcjansen) [reply test/spec/view/form/AddWms.test.js]: I have now removed 66% of these. Better? :smile:

**Pull request merged**